zero/recommendation_algorithm.py

```diff
@@ -39,13 +40,16 @@ def register(self, name, klass, default_kwargs):
 class RecommendationAlgorithm:
     factory = RecommendationAlgorithmFactory()
 
-    def __init__(self, verbose_level=1):
+    def __init__(self, metrics=['rmse'], verbose_level=1):
```

Comment:
`metrics` will be mutable and it will accumulate, didn't you want to do: `metrics=None` and `if metrics is None: metrics = ['rmse']` ?